README.md

```diff
@@ -21,13 +21,25 @@ And then download the artifact incognia-api-client
 <dependency>
   <groupId>com.incognia</groupId>
   <artifactId>incognia-api-client</artifactId>
-  <version>2.0.0</version>
+  <version>2.2.0</version>
 </dependency>
```

Comment:
I guess we should fix the repository declaration for maven, the current one only works with gradle